lockbox-ios/View/FxAView.swift

```diff
@@ -35,6 +36,7 @@ class FxAView: UIViewController, FxAViewProtocol, WKNavigationDelegate {
 
     func loadRequest(_ urlRequest: URLRequest) {
         self.webView.load(urlRequest)
+        self.url = urlRequest.url
```

Comment:
_Might_ be worth checking FxA's intention about redirects.

In fact: we should likely:

 a. Set `url` as `URL!` — because it's only going to be set once, and before it's needed.
 b. Check that navigation has finished, and iff `url` is unset, the set it — so `url` only contains the final destination of the first requested page.

Reply:
I don't totally follow what you mean with b). Where should we check that the the navigation is finished? I think `loadRequest` only gets called when the view is loaded initially.